torch/_torch_docs.py

```diff
@@ -6628,21 +6686,18 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.orgqr,
            r"""
-orgqr(input, input2) -> Tensor
+orgqr(input, tau) -> Tensor
 
-Computes the orthogonal matrix `Q` of a QR factorization, from the `(input, input2)`
+Computes the orthogonal matrix `Q` of a QR factorization, from the `(input, tau)`
 tuple returned by :func:`torch.geqrf`.
 
-This directly calls the underlying LAPACK function `?orgqr`.
-See `LAPACK documentation for orgqr`_ for further details.
+This directly calls the underlying LAPACK function `orgqr`.
+
+Alias for :func:`torch.householder_product`.
```

Comment:
Maybe we should just have the "Alias for ..." line? The rest is redundant with it.